submissions/gorodiskiy/HTML-CSS-Popup/css/main.css

```diff
@@ -91,7 +91,14 @@ input[name="open-box"]{
 input[name="open-box"]:checked~.box {
     display: block;
 }
+input[name="more"]{
+  position: absolute;
+  height:40px;
+  width:100%;
+  display: block;
+  margin: -40px 0 0 0;
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide checkbox in right way.
Bring back styles on `:focus` state.

Reply:
Yes, but how will I open the second popup then? if my input will lie inside the label, I will not be able to access ul.
Or embed ul in label too

label> input + ul

Reply:
Use this markup:
```HTML
<input id="someId">
<label for="someId"></label>
<ul>
  <li></li>
</ul>
```
```CSS
input:focused + label {
    rule: value;
}

input:checked ~ ul {
    display: block;
}
```
Hope it helps you. Read more about [sibling combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_combinator)

Reply:
Yes, just an example of Makeev, there is an input inside the label. Maybe I understood you too literally

Reply:
It doesn't matter where input in, an inside label or outside. The correct way to hide checkbox/radio button is using rules from the class `.visually-hidden` or `appearance: none;` to input.)